C++/Navio/Navio2/RGBled.cpp

```diff
@@ -14,7 +14,7 @@ bool RGBled::initialize() {
     pinR = new Pin(RPI_GPIO_4);
     pinG = new Pin(RPI_GPIO_27);
     pinB = new Pin(RPI_GPIO_6);
-    if (pinR->init() & pinG->init() & pinB->init()) {
+    if (pinR->init() && pinG->init() && pinB->init()) {
```

Comment:
The change is correct but the commit message is not good.
`C++: Navio2: RGBLed: fail on logic AND in pin initialization`

The body should say that there had been bitwise and in the past.